ex02.exs

```diff
@@ -32,26 +50,26 @@ defmodule Test do
   Replace the placeholders with your code.
   """
 
-  # test "counter using an agent" do
-  #   { :ok, counter } = « your code »
-  # 
-  #   value   = « your code »
-  #   assert value == 0
-  # 
-  #   value   = « your code »
-  #   assert value == 1
-  # end
+   test "counter using an agent" do
+     
+     { :ok, agent } = Agent.start_link(fn -> 0 end)
+     value  = Agent.get(agent, &(&1))  
+     assert value == 0
+   
+     value =  Agent.get(agent, &(&1+1)) 
```

Comment:
## -5

You aren't actually updating the value in the agent. You'll simply adding 1 to the value it returns.

You should have used 

~~~ elixir
Agent.get_and_update(agent, fn count -> { count, count + 1 } end)
~~~